packages/site/cms-site/src/image/Image.tsx

```diff
@@ -2,12 +2,21 @@
 import NextImage, { ImageLoaderProps, ImageProps as NextImageProps } from "next/image";
 
 // Fallback to 1 / 1 aspect ratio for invalid value format
-export function parseAspectRatio(value: string): number {
-    const [width = 1, height = 1] = value.split("x").map((v) => {
-        let ret: number | undefined = Number(v);
-        if (isNaN(ret)) ret = undefined;
-        return ret;
-    });
+export function parseAspectRatio(value: string | number): number {
+    let width, height;
+    if (!Number(value)) {
+        [width, height] = String(value)
+            .split(/[x/:]/)
+            .map((v) => {
+                let ret: number | undefined = Number(v);
+                if (isNaN(ret)) ret = undefined;
```

Comment:
We should probably use parseFloat instead. Otherwise `"-Infinity"` would be considered a valid aspect ratio: 
<img width="216" alt="image" src="https://github.com/user-attachments/assets/c86ae333-fa13-46c8-8e5b-354c1484a2f4">